webdriver/tests/bidi/browsing_context/locate_nodes/sandbox.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+
+from webdriver.bidi.modules.script import ContextTarget
+
+
+@pytest.mark.asyncio
+async def test_locate_nodes_in_sandbox(bidi_session, inline, top_context):
+    url = inline("""<div data-class="one">foobarBARbaz</div><div data-class="two">foobarBARbaz</div>""")
+    await bidi_session.browsing_context.navigate(
+        context=top_context["context"], url=url, wait="complete"
+    )
+
+    result = await bidi_session.browsing_context.locate_nodes(
+        context=top_context["context"],
+        locator={ "type": "css", "value": "div[data-class='one']" },
+        sandbox="sandbox"
+    )
+
+    assert len(result["nodes"]) == 1
+    node_id = result["nodes"][0]["sharedId"]
+
+    # Since the node was found in the sandbox, it should be available
+    # to scripts running in the sandbox.
```

Comment:
Actually, the `sharedId` is shared across sandboxes, while `handle` is not.
[6.5.3.17. The script.SharedId Type](https://w3c.github.io/webdriver-bidi/#type-script-SharedId):
> The script.SharedId type represents a reference to a DOM [Node](https://dom.spec.whatwg.org/#node) that is usable in any realm (including [Sandbox Realms](https://w3c.github.io/webdriver-bidi/#sandbox-realm)).

I'd assume the test was supposed to verify that the [`handle`](https://w3c.github.io/webdriver-bidi/#type-script-Handle) is provided in the given sandbox. For that, `ownership: "root"` has to be provided.

Reply:
It was not intended to (because I didn't read that part of the spec closely enough), but should be extended to include checks of the resulting values though.

Reply:
@sadym-chromium I've added a test for `handle` given `root` ownership in different sandboxes. Is that sufficient for the testing of your concern? If not, can you please give me some additional info so I can make sure the correct tests are present?